src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseStudyServiceImpl.java

```diff
@@ -1009,4 +1027,19 @@ public T findByIdInAnyTenant(final Long studyId) {
 
         return studyRepository.findByIdInAnyTenant(studyId);
     }
+
+    @Override
+    public T findWorkspaceForUser(IUser user) {
+
+       T workspace = studyRepository.findWorkspaceForUser(user.getId(), user.getActiveTenant().getId());
```

Comment:
pay attention to domain that is used in this repository, it's a view where tenant is already considered